packages/client/src/views/Home/RecordAudit.tsx

```diff
@@ -0,0 +1,608 @@
+  dateOfBirth: 'Date of birth',
+  dateOfDeath: 'Date of death',
+  placeOfBirth: 'Place of birth',
+  placeOfDeath: 'Place of death',
+  informant: 'Informant',
+  brn: 'BRN',
+  drn: 'DRN'
+}
+
+const NO_DATA_LABEL: ILabel = {
+  status: 'No status',
+  type: 'No event',
+  trackingId: 'No tracking id',
+  dateOfBirth: 'No date of birth',
+  dateOfDeath: 'No date of death',
+  placeOfBirth: 'No place of birth',
+  placeOfDeath: 'No place of death',
+  informant: 'No informant'
+}
+
+const getCaptitalizedword = (word: string | undefined): string => {
+  word = word && word.toUpperCase()[0] + word.toLowerCase().slice(1)
+  return word || ''
+}
+
+const isBirthApplication = (
+  application: GQLEventSearchSet | null
+): application is GQLBirthEventSearchSet => {
+  return (application && application.type === 'Birth') || false
+}
+
+const isDeathApplication = (
+  application: GQLEventSearchSet | null
+): application is GQLDeathEventSearchSet => {
+  return (application && application.type == 'Death') || false
+}
+
+const getDraftApplicationName = (application: IApplication): string => {
+  let name = ''
+  const applicationName = application.data.child || application.data.deceased
+  if (applicationName) {
+    if (applicationName.firstNamesEng && applicationName.familyNameEng) {
+      name = applicationName.firstNamesEng + ' ' + applicationName.familyNameEng
+    } else {
+      name =
+        (applicationName.firstNamesEng as string) ||
+        (applicationName.familyNameEng as string) ||
+        ''
+    }
+  }
+  return name
+}
+
+const getWQApplicationName = (application: GQLEventSearchSet): string => {
+  let name = ''
+  const applicationName =
+    (isBirthApplication(application) &&
+      application.childName &&
+      application.childName[0]) ||
+    (isDeathApplication(application) &&
+      application.deceasedName &&
+      application.deceasedName[0])
+
+  if (applicationName) {
+    if (applicationName.firstNames && applicationName.familyName) {
+      name = applicationName.firstNames + ' ' + applicationName.familyName
+    } else {
+      name = applicationName.familyName || applicationName.firstNames || ''
+    }
+  }
+  return name
+}
+
+const getGQLApplicationName = (application: IGQLApplication): string => {
+  let name = ''
+  const applicationName =
+    application && application.child && application.child.name
+      ? application.child.name[0]
+      : application.deceased &&
+        application.deceased.name &&
+        application.deceased.name[0]
+  if (applicationName) {
+    if (
+      (applicationName as GQLHumanName).firstNames &&
+      (applicationName as GQLHumanName).familyName
+    ) {
+      name =
+        (applicationName as GQLHumanName).firstNames +
+        ' ' +
+        (applicationName as GQLHumanName).familyName
+    } else {
+      name =
+        (applicationName as GQLHumanName).familyName ||
+        (applicationName as GQLHumanName).firstNames ||
+        ''
+    }
+  }
+  return name
```

Comment:
And here

Reply:
I think it would be good to have some sort of function for passing in the object that contains the names that returns the full name from it. We might already have something like that somewhere